src/Http/Controllers/Views/Projects/LargoController.php

```diff
@@ -36,11 +38,32 @@ public function index(Request $request, $id)
         $patchUrlTemplate = Storage::disk(config('largo.patch_storage_disk'))
             ->url(':prefix/:id.'.config('largo.patch_format'));
 
+        $shapes = collect([
+            1 => 'Point',
+            2 => 'LineString',
+            3 => 'Polygon',
+            4 => 'Circle',
+            5 => 'Rectangle',
+            6 => 'Ellipse',
+            7 => 'WholeFrame',
+        ]);
```

Comment:
Use `$shapes = Shape::pluck('name', 'id')` instead. These are database models. The (order of) IDs are not guaranteed to be the same in each application instance.